tests/test-cluster.py

```diff
@@ -24,6 +24,17 @@
 channel_to_test = os.environ.get("CHANNEL_TO_TEST", "latest/stable")
 backend = os.environ.get("BACKEND", None)
 profile = os.environ.get("LXC_PROFILE", "lxc/microk8s.profile")
+launch_config = """---
+version: 0.1.0
+extraCNIEnv:
+  IPv4_SUPPORT: true
+  IPv4_CLUSTER_CIDR: 10.3.0.0/16
+  IPv4_SERVICE_CIDR: 10.152.183.0/24
```

Comment:
Why we changed to 10.152.183.0 from 10.153.183.0?

Reply:
Same as above (https://github.com/canonical/microk8s/pull/4079#discussion_r1263715249)

Reply:
Now that we updated the core addon the 10.153.183.0 should work. Lets Try this.

Reply:
Yes trying that.